tasks/yarn.yml

```diff
@@ -47,23 +48,47 @@
     mode: g=rXw,o=rX
     recurse: true
 
+- name: set permissions to client
+  file:
+    dest: "{{ wifi_login_pages_path }}/client"
+    group: "{{ www_group }}"
+    # sets executable only to directories
+    mode: u=rwX,g=rX,o=rX
+    recurse: true
+
+- name: set permissions to server
+  file:
+    dest: "{{ wifi_login_pages_path }}/server"
+    group: "{{ www_group }}"
+    # sets executable only to directories
+    mode: u=rwX,g=rX,o=rX
+    recurse: true
+
+- name: copy organizations
+  become: true
+  copy:
+    src: organizations
+    dest: "{{ wifi_login_pages_path }}"
+    mode: 0644
```

Comment:
can you make sure the files are copied with the www user and group please?

Reply:
Done